docker/memgraph.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.8'
+services:
+  memgraph:
+    image: 'memgraph/memgraph-platform'
+    hostname: memgraph
+    ports:
+      - '3000:3000'
+      - '7444:7444'
+      - '7687:7687'
+    volumes:
+      - mg_lib:/var/lib/memgraph
```

Comment:
Are these the correct volume paths?  It looks like you added `docker/.memgraph` to the .gitignore file, but these all look like they'll be in individual folders and not within a .memgraph folder.